jest.config.js

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
A little nitpicky, but normally we add `"jest"` to the `package.json` and include this information instead of creating a `jest.config.js` file. Furthermore, for consistency, can we change `jest.setup.js` to `test-setup.js` like our other repos?

Reply:
I've added the config back in `package.json` ( 2d01ba7 )